purchases/src/main/kotlin/com/revenuecat/purchases/galaxy/IAPHelperProvider.kt

```diff
@@ -0,0 +1,10 @@
+package com.revenuecat.purchases.galaxy
+
+import com.samsung.android.sdk.iap.lib.listener.OnGetProductsDetailsListener
+
+internal interface IAPHelperProvider {
+    fun getProductsDetails(
```

Comment:
I'm wondering if we should also annotate this method with the `@GalaxySerialRequest`, so if/when we add other operation helpers, it's easier to remember we need to add that annotation there as well.